definitions/private-law/json/CaseEvent.json

```diff
@@ -351,6 +351,7 @@
     "PostConditionState": "SUBMITTED_NOT_PAID",
     "SecurityClassification": "Public",
     "CallBackURLAboutToStartEvent": "${CCD_DEF_COS_URL}/getSolicitorAndFeeDetails",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/generate-document-submit-application",
```

Comment:
Is it agreed that the documents will be generated before payment? If yes, then the changes are fine.

Reply:
yes its agreed